### DISCUSSION TITLE
Can't change upgrades modules's seed numbers

### DISCUSSION THREAD

**Discussion by MattisMatt** [zencq/NomNom#102] [Q&A] [answered]
Hello !
First thanks a lot for this awesome tool and all the work to do it.

Maybe I missed something, but I can't change any upgrade module seed number in any inventory.
Each time I try, the original number comes back, whatever I do (tried to type "enter", to hit the generate button right aside...Nothing)...
I also tried to modify the restrictions in the Nom Nom 's settings to go in "modding", in vain.

Hope somebody could help me, thanks.
Matt

**Comment by cengelha** [comment, marked as answer]
Hi Matt,
thanks for the report! This is actually a bug that will be fixed in the next version.
